csharp_dotnetcore/3.Welcome-Users/WelcomeUserBot.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Newtonsoft.Json;
+using WelcomeUser.State;
+
+namespace WelcomeUser
```

Comment:
This namespace will change, probably to "Microsoft.BotBuilderSamples". We can do that in round 2.